### PR TITLE
Add documentation for the release label for pull requests

### PR DESCRIPTION
**Pull request by simonrozsival**
What the title says.

### CODE REVIEW

File: pull-request-etiquette.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ To help us write good pull request descriptions, we use [pull request templates]
 
 It is the contributor's responsibility to make sure the merging branch is up to date with the base branch. If you would like feedback on a work-in-progress feature, feel free to create a pull request and mark it with the `wip` label. Make sure to comment on what kind of feedback you are looking for.
 
+To distinguish a release pull request from ordinary pull requests and thus make the life of reviewers a bit easier, the author should consider adding the `release` label to the pull request.
```
Comment (amulware) [inline pull-request-etiquette.md]: Why only consider? Do or do not. :)

**Comment by amulware** [comment]
That being said, I'm not sure if the PR etiquette is the best place for this. That's really about helping new contributors while doing releases is for seasoned folks. I'd include it in the list of labels, and the release documentation instead of here. What say you?

File: pull-request-etiquette.md (hunk 1 above)
Comment (kornelijepetak) [reply pull-request-etiquette.md]: I say DO!

Comment (simonrozsival) [reply pull-request-etiquette.md]: OK!

**Comment by simonrozsival** [comment]
The list of labels is clearly marked as "labels for issues". I will add it to the same doc but under a (new) PR section.

**Pull request merged**